models/ensemble.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import pickle as pkl
+from collections import Counter
+
+from metrics import metric_dict
+
+
+class EnsembleModel(object):
+    def __init__(self, model_dir, use_iterator=False):
```

Comment:
Q: Why use iterator here?

Reply:
Sometimes the models can be quite large, so loading them all into memory is intensive. This option allows for dynamic loading of the models (one-by-one) to minimize memory consumption.